src/views/pages/importGestionnaireReseauPage/ImportGestionnaireReseau.tsx

```diff
@@ -113,17 +113,27 @@ const RésultatSoumissionFormulaire: FC<RésultatSoumissionFormulaireProps> = ({
 type CsvValidationErrorBoxProps = {
   erreursDeValidationCsv: Array<ErreurValidationCsv>
 }
-const CsvValidationErrorBox: FC<CsvValidationErrorBoxProps> = ({ erreursDeValidationCsv }) => (
-  <ul className="notification error">
-    {erreursDeValidationCsv.map(({ numéroLigne, valeurInvalide, raison }, index) => (
-      <li key={index} className="ml-3">
-        {numéroLigne && `Ligne ${numéroLigne.toString()} - `}
-        {valeurInvalide && `${valeurInvalide} - `}
-        {raison && `${raison}`}
-      </li>
-    ))}
-  </ul>
-)
+const CsvValidationErrorBox: FC<CsvValidationErrorBoxProps> = ({ erreursDeValidationCsv }) => {
+  const afficherErreur = ({ numéroLigne, valeurInvalide, raison }: ErreurValidationCsv) => {
+    return `${numéroLigne ? `Ligne ${numéroLigne.toString()} - ` : ''}${
+      valeurInvalide ? `${valeurInvalide} - ` : ''
+    }${raison && `${raison}`}`
```

Comment:
Identique à ça je crois
```suggestion
    }${raison}`
```